projects/elife.yaml

```diff
@@ -68,7 +68,9 @@ defaults:
             idle_timeout: 60
             certificate: arn:aws:iam::512686554592:server-certificate/cloudfront/wildcard.elifesciences.org/wildcard.elifesciences.org
             healthcheck:
+                protocol: http
                 port: 80
+                path: /ping
```

Comment:
I would go for being conservative with `/ping`, since otherwise any backend having problems would take down journal, while we have pages that tried to degrade the experience.